atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -208,7 +208,7 @@ protected final void check() {
         }
         Preconditions.checkNotNull(keyspace(), "'keyspace' must be specified");
         double evictionCheckProportion = proportionConnectionsToCheckPerEvictionRun();
-        Preconditions.checkArgument(evictionCheckProportion > 0 && evictionCheckProportion <= 1,
+        Preconditions.checkArgument(evictionCheckProportion > 0.01 && evictionCheckProportion <= 1,
                 "'proportionConnectionsToCheckPerEvictionRun' must be between 0 and 1");
```

Comment:
Change the error message as well "'proportionConnectionsToCheckPerEvictionRun' must be between 0.01 and 1");".